timescale-forge/vpc-peering.md

```diff
@@ -0,0 +1,269 @@
+from the AWS console.
+
+<img class="main-content__illustration" src="create-vpc.png" alt="Create a new Forge VPC"/>
+
+<highlight type="tip">
+VPC peering can be enabled for free during your Timescale Forge trial, but you will be
+required to enter a valid payment method in order to create a VPC (even though you
+will not yet be charged for it).
+</highlight>
+
+### Create a peering connection
+When you have created a Timescale Forge VPC, you are ready to create a peering connection
+between your Forge VPC and your cloud VPC. To do this, expand the dropdown menu by clicking
+the `VPC Peering` column on your Forge VPC.
+
+<img class="main-content__illustration" src="create-peering-connection.png" alt="Expand the VPC Peering dropdown menu and enter info"/>
+
+From this menu, enter the AWS Account ID, AWS VPC ID, and AWS region of your
+cloud VPC for the new peering connection. When you have entered the correct
+information, click `Add peering connection` to begin the peering process.
+
+This process is asynchronous, and results in a peering connection sent to your
+AWS account for you to accept. This is an important safety mechanism, never
+accept a peering connection from an unknown account. All Timescale Forge
+peering connections are sent from the AWS account `142548018081`.
+
+Navigate to the AWS console's
+[Peering Connections](https://console.aws.amazon.com/vpc/home#PeeringConnections:)
+dashboard. Find the new peering connection request sent from Timescale Forge, and
+accept the request.
+
+<highlight type="tip">
+Make note of the peering connection ID (starting with `pcx-`) as it is used in the next step.
+</highlight>
+
+<img class="main-content__illustration" src="aws-accept-peering-connection.png" alt="Accept peering connection in AWS console after verifying requestor account number"/>
+
+### Network routing and security
+Once you have accepted the peering connection from the last step, the two VPCs will now
+be peered; however, in order to use this peering connection, you need to update your
+VPC's route table to include the CIDR block of your peered Timescale Forge VPC, and you
+also need to update your VPC's security groups.
+
+#### Route table
+Within the AWS console, navigate to the
+[Route Tables](https://console.aws.amazon.com/vpc/home#RouteTables:sort=routeTableId)
+dashboard. Select the route table corresponding to your VPC. From the detail menu, select
+the `Routes` tab and click the `Edit routes` button.
+
+<img class="main-content__illustration" src="aws-route-table-routes.png" alt="The AWS Route Tables dashboard with Routes tab expanded"/>
+
+From this view, click `Add route`. In the `Destination` column of the new row,
+enter the CIDR block of the Timescale Forge VPC for which the peering connection
+was configured in the previous few steps.
+
+In the `Target` column, enter the peering connection ID (starting with `pcx-`)
+noted in the previous step where you created the peering connection.
```

Comment:
Maybe it's me, but the "where you created" or "was configured" stuff seems superfluous and can be 🔪 . That said, in documentation we probably need to be painfully explicit

Reply:
I do think that this may be a case where we want to keep the reference to the previous step, as that previous step has a "tip" box calling out the action. Thoughts?

Reply:
Like I said, it's probably the case that documentation has to be painfully explicit as to avoid any ambiguity. I think in most cases the context makes it clear, but it's probably better just to leave it (less people sending in support 🎫 )

Reply:
Copy that. I'll leave this unresolved for now in case anyone else has a preference on this one.

Reply:
Maybe a different approach is to note somewhere near the top (or the top of the relevant sections) the items you'll need to save for later (like give them a little table that they can mentally fill in or copy to a clipboard) and then reference that as you go. It's generally helpful for me when I know ahead of time that I'm about to do something and theres a piece of information I need to save for later.

Also, for images like the "Accept Request" in AWS, it's helpful to highlight/outline the information you're referring to. It was honestly really hard for me to find the `pcx-` text in the image.

Reply:
Definitely adding screenshots will help here. We can also do things like blurring out the non-important bits of a screenshot to highlight where people need to look (that also helps with maintaining screenshots over time, too).